### ISSUE TITLE
Doesn't work with local configuration

### ISSUE THREAD

**Issue by mockdeep** [Donov4n/stylelint-find-new-rules#13]
We inherit from a local file, which causes an error with `stylelint-find-new-rules`:

```
$ stylelint-find-new-rules
Error: Invalid or unexpected token
```

Here's our `.stylelintrc.yml`:

```yml
extends:
  - stylelint-config-standard-scss
  - ./.stylelint_todo.yml
```

<details>

<summary>And here's our `.stylelint_todo.yml`:</summary>

```
# This configuration was generated by `exe/stylelint_autogen`
# on 2022-03-02 23:08:55 UTC.
# The point is for the user to remove these configuration records
# one by one as the offenses are removed from the code base.

overrides:

  # Offense count: 11
  - rules: { 'at-rule-empty-line-before': null }
    files:
    - app/assets/stylesheets/application.scss
    - app/assets/stylesheets/jquery_ui.scss
    - app/assets/stylesheets/legacy.scss

  # Offense count: 2
  - rules: { 'block-closing-brace-empty-line-before': null }
    files:
    - app/assets/stylesheets/refresh/_layouts.scss
    - app/assets/stylesheets/refresh/pages/_campaigns.scss

  # Offense count: 1
  - rules: { 'block-closing-brace-newline-before': null }
    files:
    - app/assets/stylesheets/refresh/pages/_steps.scss

  # Offense count: 5
  - rules: { 'declaration-block-no-redundant-longhand-properties': null }
    files:
    - app/assets/stylesheets/refresh/_layouts.scss
    - app/assets/stylesheets/refresh/pages/_console.scss
    - app/assets/stylesheets/refresh/pages/_forms_team.scss
    - app/assets/stylesheets/refresh/pages/_inbox.scss

  # Offense count: 1
  - rules: { 'declaration-block-trailing-semicolon': null }
    files:
    - app/assets/stylesheets/refresh/pages/_steps.scss

  # Offense count: 4
  - rules: { 'declaration-colon-newline-after': null }
    files:
    - app/assets/stylesheets/components/_fields.scss
    - app/assets/stylesheets/doc_editor/_context_menu.scss
    - app/assets/stylesheets/vendor/_fonts.scss

  # Offense count: 2
  - rules: { 'font-family-name-quotes': null }
    files:
    - app/assets/stylesheets/refresh/pages/_campaigns.scss

  # Offense count: 2
  - rules: { 'font-family-no-missing-generic-family-keyword': null }
    files:
    - app/assets/stylesheets/refresh/pages/_campaigns.scss

  # Offense count: 16
  - rules: { 'indentation': null }
    files:
    - app/assets/stylesheets/components/_fields.scss
    - app/assets/stylesheets/doc_editor/_context_menu.scss
    - app/assets/stylesheets/vendor/_fonts.scss

  # Offense count: 73
  - rules: { 'max-empty-lines': null }
    files:
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/refresh/pages/_campaigns.scss
    - app/assets/stylesheets/refresh/pages/_test_approvers.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 19
  - rules: { 'max-line-length': null }
    files:
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/mailer.css.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss
    - app/assets/stylesheets/vendor/_fonts.scss

  # Offense count: 1
  - rules: { 'no-extra-semicolons': null }
    files:
    - app/assets/stylesheets/refresh/pages/_campaigns.scss

  # Offense count: 1
  - rules: { 'property-no-unknown': null }
    files:
    - app/assets/stylesheets/mailer.css.scss

  # Offense count: 3
  - rules: { 'property-no-vendor-prefix': null }
    files:
    - app/assets/stylesheets/admin.scss
    - app/assets/stylesheets/mailer.css.scss

  # Offense count: 25
  - rules: { 'scss/at-extend-no-missing-placeholder': null }
    files:
    - app/assets/stylesheets/doc_editor/_toolbar.scss
    - app/assets/stylesheets/legacy.scss
    - app/assets/stylesheets/legacy/bootstrap_with_overrides.scss
    - app/assets/stylesheets/refresh/extensions/_bootstrap.scss
    - app/assets/stylesheets/refresh/overrides/_bootstrap.scss
    - app/assets/stylesheets/refresh/pages/_responses.scss
    - app/assets/stylesheets/wizard.scss

  # Offense count: 1
  - rules: { 'scss/at-import-partial-extension': null }
    files:
    - app/assets/stylesheets/jquery_ui.scss

  # Offense count: 1
  - rules: { 'scss/at-mixin-argumentless-call-parentheses': null }
    files:
    - app/assets/stylesheets/group_wizard/_reviewer_column.scss

  # Offense count: 88
  - rules: { 'scss/comment-no-empty': null }
    files:
    - app/assets/stylesheets/actiontext.scss
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 274
  - rules: { 'scss/dollar-variable-colon-space-after': null }
    files:
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 1
  - rules: { 'scss/dollar-variable-empty-line-before': null }
    files:
    - app/assets/stylesheets/wizard.scss

  # Offense count: 437
  - rules: { 'scss/double-slash-comment-whitespace-inside': null }
    files:
    - app/assets/stylesheets/actiontext.scss
    - app/assets/stylesheets/components/_fields.scss
    - app/assets/stylesheets/form_filler/_tooltips.scss
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/legacy/variables/_colors.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 15
  - rules: { 'scss/no-global-function-names': null }
    files:
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/legacy/variables/_colors.scss
    - app/assets/stylesheets/refresh/pages/_inbox.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 4
  - rules: { 'selector-attribute-quotes': null }
    files:
    - app/assets/stylesheets/refresh/extensions/_bootstrap.scss
    - app/assets/stylesheets/refresh/pages/_header.scss
    - app/assets/stylesheets/tree/_search_results.scss

  # Offense count: 3
  - rules: { 'selector-id-pattern': null }
    files:
    - app/assets/stylesheets/tree/campaigns/_new.scss

  # Offense count: 2
  - rules: { 'selector-no-vendor-prefix': null }
    files:
    - app/assets/stylesheets/components/_html_form_validation.scss
    - app/assets/stylesheets/tree/_main.scss

  # Offense count: 1
  - rules: { 'selector-pseudo-class-no-unknown': null }
    files:
    - app/assets/stylesheets/admin.scss

  # Offense count: 12
  - rules: { 'value-keyword-case': null }
    files:
    - app/assets/stylesheets/legacy/variables/_bootstrap.scss
    - app/assets/stylesheets/refresh/variables/_bootstrap.scss

  # Offense count: 1
  - rules: { 'value-list-comma-newline-after': null }
    files:
    - app/assets/stylesheets/vendor/_fonts.scss

  # Offense count: 1
  - rules: { 'value-no-vendor-prefix': null }
    files:
    - app/assets/stylesheets/refresh/pages/_routing.scss

rules:

  # Offense count: 24
  'alpha-value-notation': null

  # Offense count: 26
  'color-function-notation': null

  # Offense count: 76
  'declaration-empty-line-before': null

  # Offense count: 79
  'no-descending-specificity': null

  # Offense count: 88
  'number-leading-zero': null

  # Offense count: 159
  'scss/double-slash-comment-empty-line-before': null

  # Offense count: 277
  'scss/operator-no-unspaced': null

  # Offense count: 214
  'selector-class-pattern': null

  # Offense count: 275
  'string-quotes': null
```

</details>

**Comment by Donov4n** [comment]
Fixed in 4.0.0 :)

_(Repro, just in case: [test-stylelint.zip](https://github.com/Donov4n/stylelint-find-new-rules/files/8238112/test-stylelint.zip))_

**Comment by mockdeep** [comment]
Great, thanks! I'll give it a try.
